python/lsst/sims/utils/AstrometryUtils.py

```diff
@@ -190,10 +190,10 @@ def applyPrecession(ra, dec, epoch=2000.0, mjd=None):
 
     """
 
-    output = _applyPrecession(numpy.radians(ra), numpy.radians(dec),
+    output = _applyPrecession(np.radians(ra), np.radians(dec),
```

Comment:
Why call the routine in radians and then return in degrees when a method applyPrecession that works in degrees exists